lib/Twig/Extensions/Extension/Intl.php

```diff
@@ -55,12 +55,17 @@ function twig_localized_date_filter(Twig_Environment $env, $date, $dateFormat =
         'full' => IntlDateFormatter::FULL,
     );
 
+    $calendarValues = array(
+        'gregorian' => IntlDateFormatter::GREGORIAN,
+        'traditional' => IntlDateFormatter::TRADITIONAL,
```

Comment:
For all the list items in PR description should use IntlDateFormatter::TRADITIONAL and only for international Gregorian calendar should use  IntlDateFormatter::GREGORIAN.
So  IntlDateFormatter::TRADITIONAL supports all calendar listed in PR description.